workflows/kneaddata_paired.smk

```diff
@@ -17,11 +17,13 @@ rule kneaddata:
         fwd = temp(os.path.join(kneadfolder, "{sample}_kneaddata_paired_1.fastq")),
         rev = temp(os.path.join(kneadfolder, "{sample}_kneaddata_paired_2.fastq")),
         log = os.path.join(kneadfolder, "{sample}_kneaddata.log")
+     resources:
+        threads= config["rule_resources"]["kneaddata"]["threads"]
+        memory= config["rule_resources"]["kneaddata"]["memory"]
+        time= config["rule_resources"]["kneaddata"]["time"]
     run: 
-        if os.path.exists(cluster.yaml):
-            processors= {cluster.yaml(kneaddata)[processors]}
-            shell("kneaddata --input {{input.fwd}} --input {{input.rev}} --reference-db /hg37 --output {} --output-prefix {{wildcards.sample}}_kneaddata --trimmomatic /opt/conda/share/trimmomatic".format(kneadfolder))
-        else: processors= 1
+        shell("kneaddata --threads {resources.threads} --memory {resources.memory} --time {resources.time} --input {{input.fwd}} --input {{input.rev}} --reference-db /hg37 --output {} --output-prefix {{wildcards.sample}}_kneaddata --trimmomatic /opt/conda/share/trimmomatic".format(kneadfolder))
```

Comment:
```suggestion
        shell("kneaddata --threads {resources.threads} --input {{input.fwd}} --input {{input.rev}} --reference-db /hg37 --output {} --output-prefix {{wildcards.sample}}_kneaddata --trimmomatic /opt/conda/share/trimmomatic".format(kneadfolder))
```

The `kneaddata` command itself doesn't have arguments for time and memory - those are only for cluster job execution